README.rst

```diff
@@ -1,5 +1,8 @@
 Watchdog
 ========
+
+[![Build Status: Windows](https://ci.appveyor.com/api/projects/status/32r7s2skrgm9ubva/branch/master?svg=true)](https://ci.appveyor.com/project/gorakhargosh/watchdog/branch/master)
```

Comment:
The correct URL is `https://ci.appveyor.com/api/projects/status/gm3nd9f2uddakc66/branch/master?svg=true`.